R/foreign_key_constraints.R

```diff
@@ -61,6 +64,47 @@ ForeignKeyConstraints <- R6::R6Class(
         }
       }
       is_serial && is_used_as_constraint
+    },
+
+    get_network_table = function(data) {
```

Comment:
I think we should lift this out into its own function that works on a character vector of table names.  That will be much easier to test, and look less daunting than a large method

Reply:
I can break this out into another function but we need the data for line 94 where we check for null or NA columns which we can ignore when building the upload order.